csharp/src/Microsoft.ML.OnnxRuntime/RunOptions.shared.cs

```diff
@@ -129,6 +129,17 @@ public void AddRunConfigEntry(string configKey, string configValue)
             NativeApiStatus.VerifySuccess(NativeMethods.OrtAddRunConfigEntry(handle, utf8Key, utf8Value));
         }
 
+        /// <summary>
+        /// Appends the specified lora adapter to the list of active lora adapters
+        /// for this RunOptions instance. All run calls with this instant will
+        /// make use of the activates Lora Adapters
+        /// </summary>
+        /// <param name="loraAdapter">Lora adapter instance</param>
+        public void AddActiveLoraAdapter(OrtLoraAdapter loraAdapter)
```

Comment:
Is there a notion of an 'inactive' LoRA adapter? Given it's in RunOptions I would have though the 'active' was implied.